src/components/Editor/MonacoInstance/language.ts

```diff
@@ -33,7 +33,7 @@ const setupCustomLogLanguage = () => {
                     TOKEN_NAME.CUSTOM_FATAL,
                 ],
                 [
-                    /(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d{3})Z?/,
+                    /\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d{3}[+-]\d{2}:\d{2}/,
```

Comment:
To match all types of ISO8601 formats (including the strict and the loose ones):
```suggestion
                    /\d{4}-\d{2}-\d{2}(?:[T ]\d{2}:\d{2}:\d{2}(?:\.\d+)?(?:[+-]\d{2}:\d{2}|Z)?)?/,
```

Reply:
Should we add a comma as well (as an alternative to the decimal point)?

Reply:
sure

> ISO 8601:2004 - 4.2.2.4 Representations with decimal fraction
...
the  decimal fraction shall be divided from the integer part by the decimal sign specified in ISO 31-0, i.e. the comma [,] or full stop [.]. Of these, the comma is the preferred sign.

Reply:
how about
```suggestion
                    \d{4}-\d{2}-\d{2}(?:[T ]\d{2}:\d{2}:\d{2}(?:[\.,]\d+)?(?:[+-]\d{2}:\d{2}|Z)?)?,
```

Reply:
* Right, the `//` is still needed. sorry for the oversight
* Without the `:`, the format is not ISO8601, but it should not hurt to make the `:` optional.